aries-site/src/examples/templates/forms/SimpleSignUpExample.js

```diff
@@ -126,6 +126,9 @@ export const SimpleSignUpExample = () => {
         <Form
           validate="blur"
           value={formValues}
+          messages={{
+            required: 'This is a required field.',
```

Comment:
I am fine with this for now (as this is how grommet works), but a nice enhancement would be to be able to enforce this out of the box with grommet-theme-hpe or some other way. Thoughts? Seems like this opens up room for inconsistency if teams need to implement a custom required string on every form they implement.

Reply:
Added https://github.com/grommet/hpe-design-system/issues/1339 to address for discussion and prioritization.